### PR TITLE
docs(tutorial/blog): Instruct to restart the dev server after marked install

### PR DESCRIPTION
**Pull request by Drew-Daniels**
Closes: #3769

- [X] Docs

### CODE REVIEW

File: docs/tutorials/blog.md [hunk 1]
```diff
@@ -573,7 +573,7 @@ Quick note on that `invariant` for the params. Because `params` comes from the U
 
 We also have an invariant for the post. We'll handle the `404` case better later. Keep going!
 
-Now let's get that markdown parsed and rendered to HTML to the page. There are a lot of markdown parsers, we'll use "marked" for this tutorial because it's really easy to get working.
+Now let's get that markdown parsed and rendered to HTML to the page. There are a lot of markdown parsers, we'll use "marked" for this tutorial because it's really easy to get working. Please note that you will need to restart your server after installing.
 
 💿 Parse the markdown into HTML
```
Comment (dca123) [inline docs/tutorials/blog.md:579]: Could we leave the following message (or similar)  around line 585 ? This message is similar to the the once above when updating prisma. Leaving it after the installation also makes it easier to follow along !


>Now that marked has been installed, we will need to restart our server. So stop the dev server and start it back up again with `npm run dev`

**Review by dca123 — CHANGES_REQUESTED** [review]

File: docs/tutorials/blog.md (hunk 1 above)
Comment (Drew-Daniels) [reply docs/tutorials/blog.md:579]: Ah, I totally agree! Thanks for the suggestion. Definitely makes more sense to have it placed there instead. I also like the idea of using a similar instruction to the prisma one for consistency. I'll send up those updates soon 👍🏼

**Review by machour — APPROVED** [review]
Great addition since this is the first tutorial for Remix, and we want to be as clear as possible.

**Review by kentcdodds — APPROVED** [review]
Great, thanks!

**Pull request merged**